res/TensorFlowLiteRecipes/BroadcastTo_001/test.recipe

```diff
@@ -0,0 +1,25 @@
+operand {
+  name: "bc_input"
+  type: FLOAT32
+  shape { dim: 2 dim: 3 }
+}
+operand {
+  name: "bc_shape"
+  type: INT32
+  shape { dim: 3 }
+  filler { tag: "explicit" arg: "1" arg: "2" arg: "3" }
+}
+operand {
+  name: "bc_ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 2 dim: 3 }
+}
+operation {
+  type: "BroadcastTo"
+  input: "bc_input"
+  input: "bc_shape"
+  output: "bc_ofm"
+  version: 2
```

Comment:
why is this version added?

Reply:
@seanshpark 
This is because `luci-value-test` failure occurs when the value of the `version` is 1(default value). ->[error message](https://github.com/Samsung/ONE/pull/11701#issuecomment-1767390375).


In tensorflow, the version of the `builtin` op of `broadcastTo` is shown as 2 or 3, 
so I created recipe files to test both of these. :) ** [reference](https://github.com/tensorflow/tensorflow/blob/ce2d218fc88c20b1d81abb017eabd436779865e4/tensorflow/lite/kernels/register.cc#L306-L311)

Reply:
OK, thanks. Please add a comment that we have to use version 2 for this Op.